notebooks/COS/Setup/Setup.ipynb

```diff
@@ -0,0 +1,324 @@
+    "* However, there are circumstances in which you may wish to use an older context, such as when you are trying to recreate the exact steps taken by another researcher. For this, see [Section 3.2](#oldcC). \n",
+    "* Finally, you may wish to tweak the individual reference files used, such that they don't correspond to any instrument context. This is discussed in our [notebook on running the `calcos` pipeline](https://github.com/spacetelescope/COS-Notebooks/blob/master/Calcos/Calcos.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id = newcC></a>\n",
+    "## 3.1. Downloading the most recent context\n",
+    "\n",
+    "#### First, we will check the CRDS website to determine what the current context is, as it changes regularly. In your browser, navigate to [the HST CRDS homepage](https://hst-crds.stsci.edu), and you will see a page as in Fig. 3.1:\n",
+    "\n",
+    "### Fig 3.1\n",
+    "<center><img src=./figures/crds1.png width =\"900\" title=\"CRDS Homepage\"> </center>\n",
+    "\n",
+    "##### At the bottom of this page is a list of recent contexts, titled \"Context History\". Clicking the context listed with the Status \"Operational\" (circled in red in the figure) will take you to that context's page, as shown in Fig. 3.2:\n",
+    "\n",
+    "### Fig 3.2\n",
+    "<center><img src=./figures/crds2.png width =\"900\" title=\"CRDS current 'Historical References' context page\"> </center>\n",
+    "\n",
+    "##### By clicking the \"cos\" tab, (circled in red), you will be open up the tab, showing a page similar to Fig. 3.3, where you can find the current cos instrument context file: `hst_cos_<context number>.imap`. This filename is circled in red in Fig. 3.3.\n",
+    "\n",
+    "### Fig 3.3\n",
+    "<center><img src=./figures/crds3.png width =\"900\" title=\"Current COS context on the CRDS site\"> </center>\n",
+    "\n",
+    "#### Note down or copy the filename you just found."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Now we can use the command line tool to download all the files in this context\n",
+    "\n",
+    "#### From your command line, we must now set several environment variables.\n",
+    "\n",
+    "First, tell the crds command line tool where to go looking for the reference files online:\n",
+    "\n",
+    "```$ export CRDS_SERVER_URL=https://hst-crds.stsci.edu```\n",
+    "\n",
+    "Next, tell it where to save the files on your computer. As long as you are consistent, this can be wherever you like:\n",
+    "\n",
+    "```$ export CRDS_PATH=${HOME}/crds_cache```\n",
+    "\n",
+    "#### Now we can run the initial command to sync all the files we might need. *This may take a few minutes to hours, depending on your internet connection*:\n",
+    "\n",
+    "```$ crds sync --contexts hst_cos_<context number>.imap --fetch-references```\n",
```

Comment:
Just a thought for these `crds` invocations: if you create a code cell and prefix it with `!`, that tells jupyter "run the rest of this line in the shell, instead of in python".  So you *could* actually have these commands be executable.  The catch is that the `export` commands won't work - since it spawns a new shell each time those changes are immediately lost.  But you could also have the commands be ``CRDS_SERVER_URL=https://hst-crds.stsci.edu CRDS_PATH=${HOME}/crds_cache crds ...`` 

I have no strong opinion either way, but thought I'd mention it just in case that sounds like an improvement to you @nkerman !